packages/frontend-core/src/utils/websocket.js

```diff
@@ -1,6 +1,7 @@
 import { io } from "socket.io-client"
+import { SocketEvent, SocketSessionTTL } from "@budibase/shared-core"
 
-export const createWebsocket = path => {
+export const createWebsocket = (path, heartbeat = true) => {
```

Comment:
I see it now - thats cool to have an option for web socket creation with heartbeat - I imagine we'll use this for other stuff.

Reply:
Yep the generic `BaseSocket` in the server, which all the others are based off, handles this automatically. So we'll get multi-instance user management handled for free going forward which should make things much easier.